SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -18,13 +19,17 @@ class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 	GENERATED_BODY()
 
 public:
-	UPROPERTY(EditAnywhere, Config, Category = "Load Balancing")
-	TSubclassOf<UAbstractLBStrategy> LoadBalanceStrategy;
+	/** Enable running different server worker types to split the simulation. */
+	UPROPERTY(EditAnywhere, Config, Category = "Multi-Worker")
+	bool bEnableMultiWorker;
```

Comment:
Currently we have bEnableMultiWorker in WorldSettings and in the GDKSettings. I'd like to remove this one.

Reply:
UNR ticked created for this one as well.